### DISCUSSION TITLE
Opening Time Machine backup (.sparsebundle) from network attached storage NAS

### DISCUSSION THREAD

**Discussion by Rabattkarte** [unsound/hfsexplorer#37] [Q&A] [answered]
Hi,

after I saw [your discussion here](https://github.com/unsound/hfsexplorer/discussions/16), my hopes where high!

My situation is that I did frequent backups from my MacBook to a Synology NAS. On that NAS, I have a share dedicated to Time Machine. This backup is encrypted.

I can access that share on Windows and see all the files. The path in question is `\\nas\time-machine\my_machine.sparsebundle`. The NAS file system is a healthy `ext4`.

---

Now, when I'm trying to access this sparsebundle via HFSExplorer, I go as follows:

1. Click File -> Load file system from path
1. Enter the path `\\nas\time-machine\my_machine.sparsebundle` -> Click OK
1. It loads for a few seconds then asks me to enter the encryption passphrase.
1. I enter the passphrase.
1. I can select either `"EFI System Partiiton" (PARTITION_TYPE_EFI_SYSTEM)` or `"disk image" (PARTITION_TYPE_APPLE_APFS)` as the GUID Partition Table partition.
1. Selecting either throws the following error (Can't find handler for partition type "UNKNOWN"):
  - ![image](https://user-images.githubusercontent.com/16440184/202308373-f19db72f-1c35-4a87-8bf7-c875904f153f.png)

Is there any chance for me to mount that backup from my NAS? As a workaround, I have to buy an external drive, copy the NAS directory over, connect it to my Windows PC via USB and access the files that way maybe.

Thank you very much!

**Comment by unsound** [comment, marked as answer]
Hi, HFSExplorer cannot read APFS volumes, only HFS+/HFSX so that's the main problem that you're facing here. As the backup is APFS formatted it cannot be accessed with this tool.

**Reply by Rabattkarte** [reply]
Duh, stupid me. I see, this answers everything, of course. Thank you very much.
